scripts/dev/contexts/evg-private-context

```diff
@@ -116,33 +116,39 @@ fi
 echo "Setting BUILD_SCENARIO=${BUILD_SCENARIO}"
 export BUILD_SCENARIO="${BUILD_SCENARIO}"
 
-# Set REGISTRY and OPERATOR_VERSION based on build scenario
-# REGISTRY and OPERATOR_VERSION can be overridden externally if needed
+# Set REGISTRY and OPERATOR_VERSION, READINESS_PROBE_VERSION, VERSION_UPGRADE_HOOK_VERSION based on build scenario
+# REGISTRY and OPERATOR_VERSION, READINESS_PROBE_VERSION, VERSION_UPGRADE_HOOK_VERSION can be overridden externally if needed
 PATCH_REGISTRY="268558157000.dkr.ecr.us-east-1.amazonaws.com/dev"
 STAGING_REGISTRY="268558157000.dkr.ecr.us-east-1.amazonaws.com/staging"
 RELEASE_REGISTRY="quay.io/mongodb"
 case $BUILD_SCENARIO in
   "${BUILD_SCENARIO_RELEASE}")
     REGISTRY="${REGISTRY:-$RELEASE_REGISTRY}"
     OPERATOR_VERSION="${OPERATOR_VERSION:-$GIT_TAG}"
+    READINESS_PROBE_VERSION_FROM_RELEASE_JSON="$(jq -r '.readinessProbeVersion' "${PROJECT_DIR}/release.json")"
+    READINESS_PROBE_VERSION="${READINESS_PROBE_VERSION:-$READINESS_PROBE_VERSION_FROM_RELEASE_JSON}"
+    VERSION_UPGRADE_HOOK_VERSION_FROM_RELEASE_JSON="$(jq -r '.versionUpgradeHookVersion' "${PROJECT_DIR}/release.json")"
```

Comment:
is this used by the configuration code which deploys the operator? Is this env var on the operator to setup the correct version for the community deployment to use?

Reply:
Not this particular, but the `READINESS_PROBE_IMAGE` and `VERSION_UPGRADE_HOOK_IMAGE` from https://github.com/mongodb/mongodb-kubernetes/blob/fade27a7988af784a4b17785f497e22f6b4efd44/scripts/dev/contexts/root-context#L88-L89C8 are used in Helm Charts as before. Previously the version used always `version_id` or `latest`